controllers/managedocs_controller.go

```diff
@@ -895,3 +903,117 @@ func (r *ManagedOCSReconciler) own(resource metav1.Object) error {
 	}
 	return nil
 }
+
+func (r *ManagedOCSReconciler) reconcileRookCephOperatorConfig() error {
+	rookConfigMap := &corev1.ConfigMap{}
+	rookConfigMap.Name = rookConfigMapName
+	rookConfigMap.Namespace = r.namespace
```

Comment:
If its alright, I'd prefer to keep this local to this function since this is the only place that gets used, then move it to the reconciler object if it becomes necessary to do so.

Reply:
👍